packages/material-ui-lab/src/internal/pickers/PickersPopper.tsx

```diff
@@ -117,6 +126,8 @@ const PickersPopper: React.FC<PickerPopperProps & WithStyles<typeof styles>> = (
       open={open}
       anchorEl={anchorEl}
       className={clsx(classes.root, PopperProps?.className)}
+      // Prevent the page from jumping to the top. Seems to be a bug in TrapFocus.
+      disablePortal
```

Comment:
There seems to be some confilcts with the combination of **Popper** and  **TrapFocus**. Add *disablePortal* prop to the **Popper** fixes the issue that page jump to the top when open a picker.